web/src/pages/lesson1/types/example.ts

```diff
@@ -0,0 +1 @@
+export function test() {}
```

Comment:
Function doesnt have a use, consider moving it?

Reply:
I'll be adding functionality to it soon